operations/serviceoverbookinglimits.md

```diff
@@ -0,0 +1,208 @@
+| `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the service overbooking limits were updated. |
+| `ActivityStates` | array of [Activity State](_objects.md#activity-state) | optional | Whether to return only active, only deleted, or both types of record. If not specified, only active records will be returned. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned and optional Cursor for the starting point of data. |
+
+### Response
+
+```javascript
+{
+  "ServiceOverbookingLimits": [
+    {
+      "Id": "e654f217-d1b5-46be-a820-e93ba568dfac",
+      "ServiceId": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+      "FirstTimeUnitStartUtc": "2024-11-01T00:00:00Z",
+      "LastTimeUnitStartUtc": "2024-11-30T00:00:00Z",
+      "Limit": 10,
+      "CreatedUtc": "2024-01-31T10:48:06Z",
+      "UpdatedUtc": "2024-01-31T10:58:06Z",
+      "IsActive": true
+    }
+  ],
+  "Cursor": "e654f217-d1b5-46be-a820-e93ba568dfac"
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ServiceOverbookingLimits` | array of [ServiceOverbookingLimit](serviceoverbookinglimits.md#serviceoverbookinglimit) | required | Service overbooking limits of the default service. |
+| `Cursor` | string | optional | Unique identifier of the item one newer in time order than the items to be returned. If Cursor is not specified, i.e. null, then the latest or most recent items will be returned. |
+
+#### ServiceOverbookingLimit
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Id` | string | required | Unique identifier of the service overbooking limit. |
+| `ServiceId` | string | required | Unique identifier of the `Service`. |
+| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last [time unit](../concepts/time-units.md), in UTC timezone ISO 8601 format. |
+| `Limit` | integer | required | Number of overbookings allowed for the `Service`. |
+| `CreatedUtc` | string | required | Timestamp in UTC timezone ISO 8601 format when the service overbooking was created. |
+| `UpdatedUtc` | string | required | Timestamp in UTC timezone ISO 8601 format when the service overbooking was updated. |
+| `IsActive` | boolean | required | Whether the service overbooking limit is still active. |
+
+## Set service overbooking limits
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change.
+
+Adds new service overbooking limits with the specified conditions. Note this operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
```

Comment:
I would suggest putting these in a single paragraph and a slight change of wording:
```
Adds new service overbooking limits with the specified conditions. Past overbooking limits cannot be updated outside of `EditableHistoryInterval`. Care is needed to specify `FirstTimeUnitStartUtc` and `LastTimeUnitStartUtc` in the correct format - see [Datetimes](../guidelines/serialization.md#datetimes). This operation supports [Portfolio Access Tokens](../guidelines/multi-property.md).
```